backend/tarsy/services/history_service.py

```
@@ -453,6 +453,32 @@ def _get_session_summary_operation():
                     if session_overview.started_at_us and session_overview.completed_at_us:
                         total_duration_ms = (session_overview.completed_at_us - session_overview.started_at_us) // 1000
                     
+                    # Calculate token usage aggregations - prefer repository-provided session totals
```

Ruff (0.12.2)

Line too long (100 > 88)

(E501)